src/main/java/core/basesyntax/impl/ReadingCsvFileImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.CantWorkWithThisFileException;
+import core.basesyntax.service.ReadingCsvFile;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadingCsvFileImpl implements ReadingCsvFile {
+    @Override
+    public List<String> reading(String fileName) {
```

Comment:
The parameter name 'fileName' could be misleading as it suggests that only the name is required, not the path. Consider renaming it to 'filePath' to clarify that a full path is expected.